doc/cookbook/source/examples/classifier/multiclass_svm.rst

```diff
@@ -0,0 +1,37 @@
+=================================
+Multiclass Support Vector Machine
+=================================
+
+Multiclass Support Vector Machine is a multi-class classifier which uses :sgclass:`CLibSVM` to do one vs one classification. `See the SVM cookbook <http://shogun.ml/cookbook/latest/examples/classifier/kernel_svm.html>`_ for more details.
```

Comment:
Can you phrase all this a bit different. Say that SVMs can be generalise to multiple classes via different strategies. Then below, regerence the CMulticlassSVM and the different strategy classes. This can be done with any SVM.